### ISSUE TITLE
Expiry time is inconsistent with express-rate-limit

### ISSUE THREAD

**Issue by ialexi-bl** [wyattjoh/rate-limit-redis#32]
The `express-rate-limit` package uses `windowMs` option to set the interval when limit should reset. In redis terms `windowMs` is supposed to set the expiry timeout. However now this is controlled by a standalone `expiry` option in `rate-limit-redis`, so `windowMs` is basically completely ignored. So code like this:

```ts
rateLimit({
  max: 100,
  windowMs: 1000,
  store: new RedisStore({...})
})
```

is supposed to allow 100 requests per second, but since the default option for `expiry` is 60 seconds, in reality this allows 100 per _minute_ which is complete nonsense. It's also not clear how this could be fixed, because it would probably also need some changes in `express-rate-limit` package which is problematic. At least it could be written in documentation that one should omit `windowMs` option and use `expiry` instead when instantiating `rateLimit` middleware

**Comment by FranciscoMendes10866** [comment]
In the documentation of the *express-rate-limit* you have the following:

`
windowMs

Timeframe for which requests are checked/remembered. Also used in the Retry-After header when the limit is reached.

Note: with non-default stores, you may need to configure this value twice, once here and once on the store. In some cases the units also differ (e.g. seconds vs miliseconds)

Defaults to 60000 (1 minute).
`

So basically we have to have the following configuration: 

``js
const limiter = new RateLimit({
  store: new RedisStore({
    client: RedisClient,
    expiry: 15 * 60 * 1000
  }),
  max: 100,
  windowMS: 15 * 60 * 1000,
  delayMs: 0
})
```

**Comment by gamemaker1** [comment]
Hi,

I recently helped out with the Typescript rewrite (PR #43), which also implements the newer `Store` interface introduced in [`express-rate-limit` 6.0.0](https://github.com/nfriedly/express-rate-limit/releases/v6.0.0). This includes implementing [the `init` function](https://github.com/nfriedly/express-rate-limit/commit/d7685aec6ca790fef82e4006543e1ac6ad380747), which allows external stores like this one to setup themselves using the options passed to the middleware.

This eliminates the need for a separate `expiry` option (we just use `windowMs` passed to the middleware), and the `expiry` option has been removed in v3.0.0.
